Gems/ToolingConnection/gem.json

```diff
@@ -0,0 +1,21 @@
+{
+    "gem_name": "ToolingConnection",
+    "display_name": "ToolingConnection",
+    "license": "What license ToolingConnection uses goes here: i.e. Apache-2.0 or MIT",
+    "license_url": "Link to the license web site goes here: i.e. https://opensource.org/licenses/Apache-2.0 Or https://opensource.org/licenses/MIT",
+    "origin": "The name of the originator goes here. i.e. XYZ Inc.",
+    "origin_url": "The primary repo for ToolingConnection goes here: i.e. http://www.mydomain.com",
+    "type": "Code",
+    "summary": "A short description of ToolingConnection goes here.",
+    "canonical_tags": [
+        "Gem"
+    ],
+    "user_tags": [
+        "ToolingConnection"
+    ],
+    "icon_path": "preview.png",
+    "requirements": "Notice of any requirements ToolingConnection has goes here. i.e. This requires X other gem",
+    "documentation_url": "Link to any documentation of ToolingConnection goes here: i.e. https://o3de.org/docs/user-guide/gems/reference/design/white-box/",
```

Comment:
The gem.json needs to be updated with O3DE licensing and a correct summary

Reply:
Good catch, will add.